modules/RTC/RTC.js

```diff
@@ -845,4 +852,16 @@ export default class RTC extends Listenable {
             throw new Error('Data channels support is disabled!');
         }
     }
+
+    /**
+     * Indicates if the endpoint id is currently included in the last N.
+     *
+     * @param {string} id the endpoint id that we check for last N
+     * @returns {boolean} true if the endpoint id is in the last N or if we
+     * don't have data channel support, otherwise we return false
+     */
+    isInLastN(id) {
+        return !this._lastNEndpoints // lastNEndpoints not initialised yet
```

Comment:
if you initialize the attribute (we should), this can be greatly simplified as

`return this._lastNEndpoints.indexOf(id) !== -1;`